README.rst

```diff
@@ -19,6 +19,7 @@ The library has been tested to work with following devices:
 
 * BDV-N9200W
 * CMT-SX7B
+* LSPX-S2
```

Comment:
I think it's worth adding a note here that about the port you figured out, otherwise this is good to go, thanks @Cryolitia!

Reply:
added :)

any idea about discover not work?

Reply:
thx for your explain. I do have a complex network topology. I tried `upnp-client` and it's no device reply, while DLNA seems to works well. I guess that either I'm not making something in my network work, or LSPX is genuinely not responding. I'll try to investigate this in more detail when i'm free.

Reply:
Sure thing! Did you get any replies with upnp-client? If yes, then the SSDP / discovery part is working, and the device doesn't likely just support this. I'll merge this PR now, thanks!

Reply:
no response at all, so I suspect there may be a problem with my network